- Make sure private includes are exported to Cmake.
- Cf. QTBUG-37417

diff -up Qt5/qtbase/mkspecs/features/create_cmake.prf.orig Qt5/qtbase/mkspecs/features/create_cmake.prf
--- Qt5/qtbase/mkspecs/features/create_cmake.prf.orig	2014-04-10 09:57:40.470084553 +0200
+++ Qt5/qtbase/mkspecs/features/create_cmake.prf	2014-04-10 09:57:51.313924081 +0200
@@ -28,7 +28,6 @@ CMAKE_MODULE_NAME = $$cmakeModuleName($$
 
 split_incpath {
     CMAKE_ADD_SOURCE_INCLUDE_DIRS = true
-    CMAKE_NO_PRIVATE_INCLUDES = true # Don't add private includes in the build dir which don't exist
     CMAKE_SOURCE_INCLUDES = \
         $$cmakeTargetPaths($$QT_MODULE_INCLUDE_BASE $$QT_MODULE_INCLUDE_BASE/Qt$${CMAKE_MODULE_NAME})
     CMAKE_SOURCE_PRIVATE_INCLUDES = \
